RiotSharp/FileCache.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace RiotSharp
+{
+    public class FileCache : ICache
+    {
+
+        /// <summary>
+        /// Create file cache instance
+        /// </summary>
+        /// <param name="dir">Directory for the cache to store in</param>
+        public FileCache(string dir = "/cache")
+        {
+            _directory = Path.Combine(Directory.GetCurrentDirectory(), dir);
+        }
+
+        private static string _directory;
+
+        private string GetPath(string key)
+        {
+            return Path.Combine(_directory, key.GetHashCode().ToString());
+        }
+
+        private T Load<T>(string path)
+        {
+            var json = File.ReadAllText(path);
+            return JsonConvert.DeserializeObject<T>(json);
+        }
+
+        private void Store<TK, TV>(TK key, TV value, long ttlMins = 24 * 60 * 7 * 4) // A month
```

Comment:
Couldn't we type the ttl as a `TimeSpan`?

Reply:
We could, but we do not need to store seconds and such, which also enables it to use less memory.

Reply:
*Or disk

Reply:
Or you can use `TimeSpan.FromDays(30).TotalMinutes`